backend/src/main/resources/application.yaml

```diff
@@ -8,7 +8,7 @@ user-management-client:
     view-users: view-users
     manage-users: manage-users
 config:
-  vanity-hostname: localhost
+  vanity-hostname: "http://localhost:9090"
```

Comment:
Can something like `vanity-hostname: http://localhost:${server.port}` be used here as the server port is defined further down in this config?

Reply:
I think that should work. In real environment it would be the actual proxy url but it's a nice convenience for local.